akka-docs/rst/general/supervision.rst

```diff
@@ -203,25 +203,26 @@ terminates a child by way of the ``system.stop(child)`` method or sending a
 
 Delayed restarts with the BackoffSupervisor pattern
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Provided as a build-in pattern the ``akka.pattern.BackoffSupervisor`` actor implements the so-called
-*exponential backoff supervision strategy*, which can be used to death-watch an actor,
-and when it terminates try to start it again, each time with a growing time delay between those restarts.
+Provided as a built-in pattern the ``akka.pattern.BackoffSupervisor`` implements the so-called
+*exponential backoff supervision strategy*, starting a child actor again when it fails, each time with a growing time delay between restarts.
 
-This pattern is useful when the started actor fails because some external resource is not available,
+This pattern is useful when the started actor fails [#]_ because some external resource is not available,
 and we need to give it some time to start-up again. One of the prime examples when this is useful is
-when a :ref:`PersistentActor <persistence-scala>` fails with an persistence failure - which indicates that
+when a :ref:`PersistentActor <persistence-scala>` fails (by stopping) with a persistence failure - which indicates that
 the database may be down or overloaded, in such situations it makes most sense to give it a little bit of time
-to recover before the peristent actor is restarted.
+to recover before the peristent actor is started.
 
-The following Scala snippet shows how to create a backoff supervisor which will start the given echo actor
-in increasing intervals of 3, 6, 12, 24 and finally 30 seconds:
+.. [#] A failure can be indicated in two different ways; by an actor terminating or crashing.
```

Comment:
slippery wording here - technically when it crashes and stops "it terminates". Instead of terminating say "stops" here?